README.md

```diff
@@ -1,6 +1,64 @@
 # TVM RFCs
 
-This repoisitory is an evolving repo containing the new RFC process for TVM, more changes to come in next few days.
+## What is an RFC?
+[what-is-an-rfc]: #what-is-an-rfc
 
+An RFC is a “Request for Change” to the TVM project. It is a design document
```

Comment:
it would be good to specify a way to identify RFCs. e.g. Python uses PEP numbers, what is our equivalent? A strawman: `RFC <discuss forum topic #>`.

Reply:
The template provides explicit RFC numbers. I think going back and making the change completely retroactively will be hard but we should bring all RFCs into the repo with explicit integer IDs and names and refer them them that way.

Reply:
ah sorry, my bad. having a number here makes sense.

but, my question is: how do you pick that number? I think that would mean that someone proposing an RFC would then have to go look at all the outstanding PRs to discover $next_rfc_number?

Reply:
Monotonically increasing by 1, regardless of if the RFC is implemented or not. Someone would have to search outstanding PRs to find the largest value. A linter could help with this.

Reply:
I guess another option is that when the PR is ready to land, there should be a merge conflict if someone submitted another RFC PR since you initially opened the PR. so at merge time, you could renumber.

as Jared points out, we may need to undertake a migration process, and given the number of past RFCs it seems likely that the migration process would overlap with brand new RFCs being submitted. this means that RFC numbers may not monotonically increase with the order of their adoption. so i'm wondering if making the RFC numbers start at 1 and then having to map to discussion forum topic ID is really buying us anything. should we just adopt the topic id from the forum into the RFC ID? then it would be simple to move back and forth.

the PR ID is yet another id that may confuse ppl, but I don't see a way around that.

this also brings up a related question. now we are discussing this on the PR rather than in the forum thread. admittedly, the discussion is about a process related to the RFC repo itself....but we are still splitting the discussion. Someone who just views the RFC for RFCs thread may not see this.

Do we have policy about where discussion should happen? splitting discussion into two parts seems bad, but code-review naturally lends to discussion because of the inline-comment feature.

Reply:
I'm flexible on numbering. The standard across many open source projects (including Rust and Kubernetes, for example), is to just increase by 1 and resolve conflicts as they arise. One problem that might come out of linking to the discussion post or pull request number is it ties the repository to technology that the project might not be using in the future.

The workflow seems to be:
* Initial discussion on Discuss, with a decision to formalize as a pull request.
* Pull request, which has more of a final review before merging.
* Upon merging, merged RFC becomes source of truth.

Reply:
Regarding old RFCs, I'm not sure there's a will to grab old RFCs and post them back up here. If we do, for documentation and other reasons, the two options for dealing with it are:

A1: Start current RFCs as some larger number, like 0100, to allow for backfilling old RFCs
A2: When adding old RFCs, add a prefix indicating that it is a legacy change, for example L0001, L0002, etc.

Reply:
ok, this seems reasonable to me. I agree with prefixing old RFCs with a letter to make it clear.